airbyte-integrations/connectors/source-blob/source_blob/spec.json

```diff
@@ -0,0 +1,108 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Blob Source Spec",
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["dataset_name", "provider", "format", "path"],
+    "properties": {
+      "dataset_name": {
+        "type": "string",
+        "description": "Name of the final table (should include only letters, numbers, dash and underscores. Note that this may be altered according to destination.)"
+      },
+      "provider": {
+        "type": "object",
+        "description": "Storage Provider or Location of the file(s) to be replicated.",
+        "oneOf": [
+          {
+            "title": "S3: Amazon Web Services",
+            "required": ["storage", "bucket"],
+            "properties": {
+              "storage": {
+                "type": "string",
+                "enum": ["S3"],
+                "default": "S3"
+              },
+              "bucket": {
+                "type": "string",
+                "description": "Name of the S3 bucket where the file(s) exist."
+              },
+              "aws_access_key_id": {
+                "type": "string",
+                "description": "In order to access private Buckets stored on AWS S3, this connector would need credentials with the proper permissions. If accessing publicly available data, this field is not necessary."
+              },
```

Comment:
Should the access key id be an airbyte secret as well? Not 100% necessary, but it does not hurt to be extra secure.

Reply:
addressed